docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -2,16 +2,12 @@
 Getting Started with Galaxy
 ===========================
 
-Galaxy workflow support in Dockstore is new and a preview of
-more full featured support to come.
+Dockstore supports Galaxy workflows and the :doc:`Launch with Galaxy <../../launch-with/galaxy-launch-with>` tutorial elaborates different ways to launch a Galaxy workflow. Please note that Galaxy workflows cannot be launched by the Dockstore CLI.
 
```

Comment:
Here, I would add two sections: 

Section: Using Galaxy Servers (such as usegalaxy.org)
Institutions across the world maintain Galaxy instances that are often shared with the greater community for free, often subsidized by grants. These are great ways to learn and use Galaxy, etc. etc...

Section: Creating a Custom Galaxy in the Cloud with Terra, AnVIL, and BioData Catalyst
Galaxy is now integrated into the Terra cloud workspace and this provides Galaxy users the ability to create a custom environment that they can scale to their needs. Additionally, the Terra workspace is able to access a rich data corpus hosted in the cloud, including many controlled-access datasets. 

Something like that? Maybe expand a bit.